apps/vanilla/src/global/form.css

```
@@ -0,0 +1,82 @@
+@import url("../global/index.css");
+
+.form {
+  display: flex;
+  flex-direction: column;
+  background: var(--form-background);
+}
+
+.form__label {
+  display: flex;
+  flex-direction: column;
+  margin: 10px 0;
+}
+
+.form__label-text {
+  color: var(--form-default-color);
+  font-size: var(--default-text-size);
+  padding-left: 10px;
+}
+
+.form__input {
+  background: none;
+  border: none;
+  color: var(--form-default-color);
+  font-size: var(--default-text-size);
+  border-bottom: 2px solid var(--form-default-color);
+  padding-left: 10px;
+  min-height: 35px;
+}
+
+.form__button {
+  color: var(--form-default-color);
+  font-size: var(--default-text-size);
+  background: none;
+  background-color: var(--form-background);
+  border: 1px solid var(--form-default-color);
+  border-radius: 5px;
+  min-height: 35px;
+  margin: 10px 0;
+  cursor: pointer;
+}
+
+.form__button:hover {
+  color: var(--form-active-color);
+  border-color: var(--form-active-color);
+}
+
+.form__input:focus {
+  border-bottom: 2px solid var(--form-active-color);
```

That way, if you change border width, you wouldn't need to change it everywhere.
```suggestion
  border-bottom-color: var(--form-active-color);
```